### PR TITLE
[DOCS] Align tabs in 'Install from PyPI' article - for master

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
This PR addresses the issue: https://github.com/openvinotoolkit/openvino/issues/15090

The tabs for `Linux and macOS` and `Windows` are linked.

### CODE REVIEW

File: docs/install_guides/installing-openvino-pip.md [hunk 1]
```diff
@@ -1,107 +1,142 @@
...
 
 Use the following command:
-```sh
-python -m pip install --upgrade pip
-```
 
-### Step 4. Install the Package
+.. code-block: sh
+
+   python -m pip install --upgrade pip
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-pip.md:75]: it's not rendered well.
at least in Safari I don't see this command
Screenshot 2023-03-03 at 7 38 28 PM

Comment (ilya-lavrenov) [reply docs/install_guides/installing-openvino-pip.md:75]: the issue is still here

Comment (totoka-intel) [reply docs/install_guides/installing-openvino-pip.md:75]: The issue also reproduces on Windows 11 Enterprise for the Chrome Version 110.0.5481.178 and Version 111.0.5563.65

File: docs/install_guides/installing-openvino-pip.md [hunk 2]
```diff
@@ -1,107 +1,142 @@
...
+
+.. code-block: sh
+
+   python -c "from openvino.runtime import Core"
+
 
 If installation was successful, you will not see any error messages (no console output).
 
-Congratulations! You finished installing OpenVINO Runtime. Now you can start exploring OpenVINO's functionality through Jupyter Notebooks and sample applications. See the <a href="#whats-next">What's Next</a> section to learn more!
+Congratulations! You finished installing OpenVINO Runtime. Now you can start exploring OpenVINO's functionality through Jupyter Notebooks and sample applications. See the :ref:`What's Next <whats-next>`section to learn more!
```
Comment (totoka-intel) [inline docs/install_guides/installing-openvino-pip.md:100]: This part is not rendered correctly on Windows 11 Enterprise Chrome Version 111.0.5563.65 and macOS Catalina version 10.15.7 Safari 15.6.1 (15613.3.9.1.16, 15613):
image

File: docs/install_guides/installing-openvino-pip.md (hunk 1 above)
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-pip.md:75]: It's been fixed :)

File: docs/install_guides/installing-openvino-pip.md (hunk 2 above)
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-pip.md]: Fixed :)

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**